### DISCUSSION TITLE
Documentation for ARG IMAGE= in dockerfile manager

### DISCUSSION THREAD

**Discussion by wwuck** [renovatebot/renovate#28916] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

37.349.2

### Please tell us more about your question or problem

I saw this comment https://github.com/renovatebot/renovate/pull/28834#discussion_r1590216507 on a recent PR so I thought I would be able to remove some of my existing regex manager configurations.

eg.
```Dockerfile
ARG IMAGE=node:12@sha256:6e5264cd4cfaefd7174b2bc10c7f9a1c2b99d98d127fc57a802d264da9fb43bd
FROM ${IMAGE}
```

I can't find any reference to this pattern anywhere in renovate docs to show what the expected syntax format should be in my Dockerfile.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Code: https://github.com/renovatebot/renovate/blob/96f760a36f489c4b1aeb4218a69009d557f0a014/lib/modules/manager/dockerfile/extract.ts#L314-L333

Example test: https://github.com/renovatebot/renovate/blob/96f760a36f489c4b1aeb4218a69009d557f0a014/lib/modules/manager/dockerfile/extract.spec.ts#L764-L782

Docs: https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/dockerfile/readme.md

**Reply by wwuck** [reply]
Thanks.  I can't see anything relevant on that docs link but the regex reference in extract.ts should be enough for testing my configuration.

**Reply by rarkins** [reply]
Sorry, I meant that's where the docs should be added

**Reply by HonkingGoose** [reply]
I created this issue to track the docs improvements. As mentioned in the issue, I'll let someone else make the improvements:

- #29872
